python/src/vmaf/core/noref_feature_extractor.py

```diff
@@ -384,8 +383,8 @@ def mscn_extract_niqe(cls, img, patch_size, mode):
 
         for j in range(0, h-block_h+1, shift_h):
             for i in range(0, w-block_w+1, shift_w):
-                m_patch1 = m_image1[j  :(j+block_h),   i  :(i+block_w)  ]
-                m_patch2 = m_image2[j/2:(j+block_h)/2, i/2:(i+block_w)/2]
+                m_patch1 = m_image1[j:j + block_h, i:i + block_w]
+                m_patch2 = m_image2[j // 2:(j + block_h) // 2, i // 2:(i + block_w) // 2]
```

Comment:
`/` yields floats in python3
One should use explicitly integer division `//` where an integer is desired (works in both py2/py3)